core/src/test/resources/flows/valids/trigger-flow-listener-namespace-condition-fail.yaml

```diff
@@ -0,0 +1,21 @@
+id: trigger-flow-listener-namespace-condition-fail
+namespace: io.kestra.tests.trigger.fail
```

Comment:
Do we still need to change the namespace here ?

Reply:
I was surprised, but yes. I still had false trigger in this specific test